Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -1329,7 +1329,9 @@ void Renderer::Swap(u32 xfb_addr, u32 fb_width, u32 fb_stride, u32 fb_height, u6
       {
         // Remove stale EFB/XFB copies.
         g_texture_cache->Cleanup(m_frame_count);
-        Core::Callback_FramePresented();
+        double LastSpeed =
+            1.0 / (m_fps_counter.GetDeltaTime() * VideoInterface::GetTargetRefreshRate());
```

Comment:
The denominator should probably be checked to ensure it is non-zero before dividing to make this bulletproof (in case there are timer precision issues for example).

Reply:
That is something I thought about but deemed extremely unlikely (I think I checked both functions and was quite sure they couldn't return 0), but the main problem is, what do we set LastSpeed to if there is a division by 0?